docs/content/latest/api/ysql/extensions.md

```diff
@@ -189,32 +232,34 @@ You'll see a result like this:
   8 |   991.22
   9 |   987.95
  10 |   996.57
+(10 rows)
 ```
-Each time you repeat the test, you'll see different generated values for _"v"_. The section [Analyzing a normal distribution with percent_rank(), cume_dist() and ntile()](../exprs/window_functions/analyzing-a-normal-distribution/), within the major documentation section [Window functions](../exprs/window_functions/), uses `normal_rand()`. It populates a table with a large number, say 100,000, rows and displays the outcome as a histogram that clearly shows the familiar bell-curve shape.
 
-These other functions are brought by `tablefunc`: `connectby()`; and `crosstab()`and  `crosstabN()`.
+Every time you repeat the test, you'll see different generated values for `v`. The section [Analyzing a normal distribution with percent_rank(), cume_dist() and ntile()](../exprs/window_functions/analyzing-a-normal-distribution/), within the major documentation section [Window functions](../exprs/window_functions/), uses `normal_rand()`. It populates a table with a large number, say 100,000, rows and displays the outcome as a histogram that clearly shows the familiar bell-curve shape.
+
+These other functions are brought by `tablefunc`: `connectby()`; and `crosstab()` and `crosstabN()`.
 
 The `connectby()` function displays a hierarchy of the kind that you see in an _"employees"_ table with a reflexive foreign key constraint where _"manager_id"_ refers to _"employee_id"_. Each next deeper level in the tree is indented from its parent following the well-known pattern.
 
 The `crosstab()`and  `crosstabN()` functions produce “pivot” displays. The _"N"_ in crosstabN() indicates the fact that a few, `crosstab1()', `crosstab2()', `crosstab3()`, are provided natively by the extension and that you can follow documented steps to create more.
 
-For more information see [`tablefunc`](https://www.postgresql.org/docs/11/tablefunc.html) in the PostgreSQL Docs.
+For more information, refer to [`tablefunc`](https://www.postgresql.org/docs/11/tablefunc.html) in the PostgreSQL docs.
```

Comment:
Should we use documentation instead of docs? (again, more formal style)

Reply:
Nah...